psi4/driver/qcdb/libmintsbasisset.py

```diff
@@ -858,8 +858,31 @@ def construct(cls, parser, mol, key, deffit=None, basstrings=None, return_atomli
                 (bastitle, basgbs, postfunc) = bas
 
                 filename = cls.make_filename(basgbs)
-                # -- First seek bas string in input file strings
-                if filename[:-4] in seek['strings']:
+
+                # If name is prefixed with "bse:", then load from basis set exchange library
+                if bastitle.lower().startswith('bse:'):
+                    if not qcel.util.which_import('basis_set_exchange', return_bool=True):
+                        raise ModuleNotFoundError("Python module 'basis_set_exchange' not found. Solve by installing it: `conda install -c conda-forge basis-set-exchange` or `pip install basis-set-exchange`")
```

Comment:
```suggestion
                        raise ModuleNotFoundError("Python module 'basis_set_exchange' not found. Solve by installing it: `conda install -c conda-forge basis_set_exchange` or `pip install basis_set_exchange`")
```
right?

Reply:
Oops. Pip regularizes it so it doesn't matter, but conda seems to insist on underscores.